# AttCovEuler block
# ROS message header
Header       header

# SBF block header including time header
BlockHeader  Block_Header   

uint8        Error   
float32      Cov_HeadHead
float32      Cov_PitchPitch
float32      Cov_RollRoll
float32      Cov_HeadPitch
float32      Cov_HeadRoll
float32      Cov_PitchRoll